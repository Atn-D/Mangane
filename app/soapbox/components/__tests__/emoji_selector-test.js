import React from 'react';
import renderer from 'react-test-renderer';
import EmojiSelector from '../emoji_selector';

describe('<EmojiSelector />', () => {
  it('renders correctly', () => {
    const component = renderer.create(<EmojiSelector />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
