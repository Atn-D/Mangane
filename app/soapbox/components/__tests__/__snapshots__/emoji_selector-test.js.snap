// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmojiSelector /> renders correctly 1`] = `
<div
  className="emoji-react-selector-container"
  onBlur={[Function]}
  onFocus={[Function]}
  tabIndex="-1"
>
  <div
    className="emoji-react-selector"
    onBlur={[Function]}
  >
    <button
      className="emoji-react-selector__emoji"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<img draggable=\\"false\\" class=\\"emojione\\" alt=\\"👍\\" title=\\":+1:\\" src=\\"[object Object]\\" />",
        }
      }
      onClick={[Function]}
      onKeyDown={[Function]}
      tabIndex={-1}
    />
    <button
      className="emoji-react-selector__emoji"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<img draggable=\\"false\\" class=\\"emojione\\" alt=\\"❤\\" title=\\":heart:\\" src=\\"[object Object]\\" />",
        }
      }
      onClick={[Function]}
      onKeyDown={[Function]}
      tabIndex={-1}
    />
    <button
      className="emoji-react-selector__emoji"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<img draggable=\\"false\\" class=\\"emojione\\" alt=\\"😆\\" title=\\":laughing:\\" src=\\"[object Object]\\" />",
        }
      }
      onClick={[Function]}
      onKeyDown={[Function]}
      tabIndex={-1}
    />
    <button
      className="emoji-react-selector__emoji"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<img draggable=\\"false\\" class=\\"emojione\\" alt=\\"😮\\" title=\\":open_mouth:\\" src=\\"[object Object]\\" />",
        }
      }
      onClick={[Function]}
      onKeyDown={[Function]}
      tabIndex={-1}
    />
    <button
      className="emoji-react-selector__emoji"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<img draggable=\\"false\\" class=\\"emojione\\" alt=\\"😢\\" title=\\":cry:\\" src=\\"[object Object]\\" />",
        }
      }
      onClick={[Function]}
      onKeyDown={[Function]}
      tabIndex={-1}
    />
    <button
      className="emoji-react-selector__emoji"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<img draggable=\\"false\\" class=\\"emojione\\" alt=\\"😩\\" title=\\":weary:\\" src=\\"[object Object]\\" />",
        }
      }
      onClick={[Function]}
      onKeyDown={[Function]}
      tabIndex={-1}
    />
  </div>
</div>
`;
